public/style.css

```diff
@@ -58,9 +61,8 @@ a {
 /* === Content === */
 
 .content {
-  margin: 0 auto;
-  max-width: 1100px;
-  padding: 30px 24px;
+  max-width: fit-content;
```

Comment:
Why remove the max-width? This makes the content extend to the full width which creates a lot of tension.